examples/src/bin/composite_template.rs

```diff
@@ -2,15 +2,10 @@
 //!
 //! This sample demonstrates how to create a widget using GTK's composite templates.
 
-#[macro_use]
-extern crate glib;
-#[macro_use]
-extern crate gtk;
-
 use gio::{prelude::ApplicationExtManual, ApplicationExt};
```

Comment:
And here `use gio::prelude::*`

The reason is that things might move between different extension traits but they'll always be part of the prelude. We don't want to change all the code everywhere whenever that happens.